CPAC/pipeline/cpac_pipeline.py

```diff
@@ -650,12 +650,11 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
                 if c.pipeline_setup['working_directory'][
                     'remove_working_dir']:
                     try:
-                        raise Exception('not happenin')
```

Comment:
@sgiavasis do you remember what [this](https://github.com/FCP-INDI/C-PAC/blame/dbe7b33300a9360aa8fd9a91bbd6aa9cf8613964/CPAC/pipeline/cpac_pipeline.py#L653) was for?

Reply:
Just debugging :)